<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>22</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>403</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>车辆管理</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>403</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="car">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QToolBox" name="toolBox">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>416</width>
              <height>551</height>
             </rect>
            </property>
            <attribute name="label">
             <string>出售车辆</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>厂家：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QComboBox" name="comboBoxFactory"/>
             </item>
             <item row="0" column="3" colspan="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>110</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>品牌：</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="2">
              <widget class="QComboBox" name="comboBoxBrand"/>
             </item>
             <item row="1" column="3" colspan="2">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>110</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>报价：</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEditPrice"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>数量：</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="spinBox"/>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>剩余车辆：</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>金额：</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="3">
              <widget class="QLineEdit" name="lineEditTotal">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="4">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>万</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="buttonSure">
                 <property name="text">
                  <string>确定</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="buttonCancle">
                 <property name="text">
                  <string>取消</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="3">
              <widget class="QLabel" name="labelLast">
               <property name="text">
                <string>20</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>416</width>
              <height>551</height>
             </rect>
            </property>
            <attribute name="label">
             <string>新车入库</string>
            </attribute>
           </widget>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="calc"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>43</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>车辆管理</string>
    </property>
    <addaction name="actionCar"/>
    <addaction name="actionCalc"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>系统设置</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCar">
   <property name="text">
    <string>车辆管理</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
